app/models/ability.rb

```diff
@@ -23,6 +25,8 @@ def initialize(user)
 
       return unless user.role&.super_admin?
       can :manage, :all
+    elsif user.mentor?
```

Comment:
We'll want all persisted users to drop into this conditional, instead of qualifying it with an `elsif`.  So for example, 

```ruby
else
  return unless user.mentor?
  can :read, Request
```

Once we implement roles, we will take another look at this.  But for now, we are only concerned with mentors.

Reply:
This, too